dev/tools/controllerbuilder/template/apis/types.go

```diff
@@ -36,13 +36,12 @@ var {{ .Kind }}GVK = GroupVersion.WithKind("{{ .Kind }}")
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
 // {{ .Kind }}Spec defines the desired state of {{ .Kind }}
-{{- if .KindProtoTag }}
-// +kcc:proto={{ .KindProtoTag }}
```

Comment:
We want this tag, don't we?  Otherwise we won't generate a mapper...

Reply:
resolved per offline discussion